### PR TITLE
Tweak: Docs - Eye-Dropper [ED-3520]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

File: docs/modules/elements-color-picker/assets/js/editor/component.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+- `$e.run( 'elements-color-picker/enter-preview' )` - Show the user a UI preview of the currently hovered color.
+  
+
+- `$e.run( 'elements-color-picker/exit-preview' )` - Exit the preview mode.
+  
+
+- `$e.run( 'elements-color-picker/apply' )` - Apply & Save the selected color on click.
+  
+
+- `$e.run( 'elements-color-picker/end' )` - End the color picking process and return to the normal editor state.
```
Comment (matipojo) [inline docs/modules/elements-color-picker/assets/js/editor/component.md:32]: Add the short description  also on top of each file

**Review by matipojo — CHANGES_REQUESTED** [review]
Nice!

File: docs/modules/elements-color-picker/assets/js/editor/component.md (hunk 1 above)
Comment (StyleShit) [reply docs/modules/elements-color-picker/assets/js/editor/component.md:32]: Done.

File: docs/modules/elements-color-picker/assets/js/editor/component.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+## Attention Needed / Known Issues:
+
```
Comment (matipojo) [inline docs/modules/elements-color-picker/assets/js/editor/component.md:36]: Please explain why we didn't implement a "pixel" picker for images.
how we handle "global color" picking.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: docs/modules/elements-color-picker/assets/js/editor/component.md (hunk 2 above)
Comment (StyleShit) [reply docs/modules/elements-color-picker/assets/js/editor/component.md:36]: Good idea!
Done.

File: docs/modules/elements-color-picker/assets/js/editor/component.md [hunk 3]
```diff
@@ -42,5 +42,12 @@ This is the JS component which is responsible for registering the following comm
 
 	removing its `nsEvents` property.
 
+
+- Our initial thought was creating a per-pixel eyedropper, but we dropped this idea (pun intended) since the only way to do that is by
+
+	taking a screenshot of the page and converting to Canvas, which brings a huge performance penalty with it.
+
+	In addition, taking a screenshot of a website is prohibited by the privacy rules of WordPress, so even if the performance would've been OK, we would still have this privacy issue.
+
```
Comment (matipojo) [inline docs/modules/elements-color-picker/assets/js/editor/component.md:50]: Remove. it's in the user browser so there is no privacy issue with it.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: docs/modules/elements-color-picker/assets/js/editor/component.md (hunk 3 above)
Comment (StyleShit) [reply docs/modules/elements-color-picker/assets/js/editor/component.md]: Done.

**Pull request merged**